cmd/sonobuoy/app/gen_plugin.go

```diff
@@ -59,6 +60,10 @@ func NewCmdGenPlugin() *cobra.Command {
 		&genPluginOpts.Paths, "paths", "p", []string{".", "./examples/plugins.d/"},
 		"the paths to search for the plugins in. Defaults to . and ./plugins.d/",
 	)
+	cmd.Flags().StringVar(
```

Comment:
This is just on the hidden, dev command which executes the plugin templates. This normally happens on the aggregator server and is not locally easy to do. I think this command though should either (1) go away (2) become the actual path by which the aggr server generates the yaml (3) go away due to frontloading the yaml generation via #638